src/index.js

```diff
@@ -60,6 +60,10 @@ export {default as default} from './react/deckgl';
 export {default as ViewportController} from './react/controllers/viewport-controller';
 export {default as MapController} from './react/controllers/map-controller';
 
+// Experimental Pure JS (non-React) bindings
+import {default as DeckGLJS} from './pure-js/deck';
```

Comment:
What's rationale before changing these names?
I guess exporting as DeckGLJS is to distinguish from the original DeckGL from ./react/deckgl.js.
but why changing DeckGL to Deck in the new ./pure-js/deck.js?

Reply:
> What's rationale before changing these names?
> I guess exporting as DeckGLJS is to distinguish from the original DeckGL from ./react/deckgl.js.

Yes. I don't like having two things called exactly the same name (goes for both global symbols and filenames). It tends to confuse both humans and computers.

> but why changing DeckGL to Deck in the new ./pure-js/deck.js?

Good question. I could call them `DeckGLJS` and `deckgljs.js`, I just didn't think those names are good, and would be changed asap. 

We need to come up with a naming scheme to separate the PureJS components and the React components in a logical way.